api-server.py

```diff
@@ -63,21 +66,30 @@ def custom_background_code():
 def main():
     parser = argparse.ArgumentParser()
 
-    group = parser.add_mutually_exclusive_group(required=True)
-    group.add_argument("-m", "--mainnet", action="store_true", default=False,
-                       help="Use MainNet instead of the default TestNet")
-    group.add_argument("-t", "--testnet", action="store_true", default=False,
-                       help="Use TestNet instead of the default TestNet")
-    group.add_argument("-p", "--privnet", action="store_true", default=False,
-                       help="Use PrivNet instead of the default TestNet")
-    group.add_argument("--coznet", action="store_true", default=False,
-                       help="Use the CoZ network instead of the default TestNet")
-    group.add_argument("-c", "--config", action="store", help="Use a specific config file")
-
-    parser.add_argument("--port-rpc", type=int, help="port to use for the json-rpc api (eg. 10332)")
-    parser.add_argument("--port-rest", type=int, help="port to use for the rest api (eg. 80)")
-
+    # Network options
+    group_network_container = parser.add_argument_group(title="Network options")
+    group_network = group_network_container.add_mutually_exclusive_group(required=True)
+    group_network.add_argument("--mainnet", action="store_true", default=False, help="Use MainNet")
+    group_network.add_argument("--testnet", action="store_true", default=False, help="Use TestNet")
+    group_network.add_argument("--privnet", action="store_true", default=False, help="Use PrivNet")
+    group_network.add_argument("--coznet", action="store_true", default=False, help="Use CozNet")
+    group_network.add_argument("--config", action="store", help="Use a specific config file")
+
+    # Ports for RPC and REST api
+    group_modes = parser.add_argument_group(title="Mode(s)")
+    group_modes.add_argument("--port-rpc", type=int, help="port to use for the json-rpc api (eg. 10332)")
+    group_modes.add_argument("--port-rest", type=int, help="port to use for the rest api (eg. 80)")
+
+    # Advanced logging setup
+    group_logging = parser.add_argument_group(title="Logging options")
+    group_logging.add_argument("--logfile", action="store", type=str, help="Logfile")
+    group_logging.add_argument("--syslog", action="store_true", help="Log to syslog instead of to log file ('user' is the default facility)")
+    group_logging.add_argument("--syslog-local", action="store", type=int, choices=range(0, 7), metavar="[0-7]", help="Log to a local syslog facility instead of 'user'. Value must be between 0 and 7 (e.g. 0 for 'local0').")
```

Comment:
👍

I'm a relative newbie to python, so not as familiar with these standard utilities 😁